### ISSUE TITLE
Screen reader is installed in a Temp folder on Windows and after some time, the OS removes it

### ISSUE THREAD

**Issue by AnyMa** [guidepup/setup#25]
# Issue

Setup:

- Node version: v16.20.2
- Guidepup version: 0.19.1
- OS Platform: Windows
- OS Release: 10 Pro

## Details

I noticed that the command "npx @guidepup/setup" installs the screen reader into a temp folder on Windows. Then, sometimes Windows runs the garbage collector (or something similar) and the screen reader is gone. Is there any way to install it in a non-temp folder, so I don't need to re-install it?

**Comment by jlp-craigmorten** [comment]
Have introduced a new flag `--nvda-install-dir` where you can pass your own directory for where NVDA should be installed to.

Let me know if have any issues!

**Comment by cmorten** [comment]
Any valid path for your device, e.g. for the tests in this repo I have:

```bash
npx @guidepup/setup --ci --record --nvda-install-dir %userprofile%\nvda
```

REF: https://github.com/guidepup/setup/blob/main/package.json#L30

Which places NVDA in the `C:\Users\runneradmin\nvda` directory in CI (you can see in the logs at https://github.com/guidepup/setup/actions/runs/7528145294/job/20489731536)

**Comment by AnyMa** [comment]
that's odd... couldn't make it work using this command:
npx @guidepup/setup --ci --record --nvda-install-dir C:\Users\xxxxxxxxxx\guidepup

**Comment by cmorten** [comment]
Let me take another look!

**Comment by cmorten** [comment]
Hmm, you might have to elaborate on your setup as not encountering any issues...

I'm on Windows 11 Home and have tried:

**Git Bash**

```bash
npx @guidepup/setup --nvda-install-dir C:\\Users\\Craig\\nvda-bash
```

> Note: Care needed with the escaping of the path otherwise the `\` will be considered an escape character.

![](https://github.com/guidepup/setup/assets/11313985/10086df0-cdb2-4bf5-a3bc-df618ed7fe77)

![](https://github.com/guidepup/setup/assets/11313985/8dad50dc-c3a7-43b3-9e04-8f75d096323d)

**Powershell**

```pwsh
 npx @guidepup/setup --nvda-install-dir C:\Users\Craig\nvda-pwsh
```

![](https://github.com/guidepup/setup/assets/11313985/618f8abf-4459-4fc3-a4c9-cf89c17914ee)

![](https://github.com/guidepup/setup/assets/11313985/435177d0-e9b9-4752-9dd0-a6f6df68ebb6)


**CMD**

```cmd
 npx @guidepup/setup --nvda-install-dir C:\Users\Craig\nvda-pwsh
```

![](https://github.com/guidepup/setup/assets/11313985/e540c778-e660-49c0-912e-677c516c4dc8)

![](https://github.com/guidepup/setup/assets/11313985/960ad1b7-4306-44f5-a767-a08a5f1d97e5)

**Comment by AnyMa** [comment]
Awesome! Thanks, Craig!
